api/node_pools.go

```diff
@@ -103,3 +104,15 @@ type NodePool struct {
 type NodePoolSchedulerConfiguration struct {
 	SchedulerAlgorithm SchedulerAlgorithm `hcl:"scheduler_algorithm,optional"`
 }
+
+// ListJobs is used to list all the jobs in a node pool.
+func (n *NodePools) ListJobs(poolName string, q *QueryOptions) ([]*JobListStub, *QueryMeta, error) {
+	var resp []*JobListStub
+	qm, err := n.client.query(
+		fmt.Sprintf("/v1/node/pool/%s/jobs", url.PathEscape(poolName)),
+		&resp, q)
+	if err != nil {
+		return nil, nil, err
+	}
+	return resp, qm, nil
+}
```

Comment:
It would be nice to keep this up there along with the other methods, or maybe move the structs up?